iocore/net/UnixNetAccept.cc

```diff
@@ -401,16 +418,18 @@ NetAccept::acceptFastEvent(int event, void *ep)
   int loop               = accept_till_done;
 
   do {
-    if (!opt.backdoor && check_net_throttle(ACCEPT)) {
-      ifd = NO_FD;
-      return EVENT_CONT;
-    }
-
     socklen_t sz = sizeof(con.addr);
     int fd       = socketManager.accept4(server.fd, &con.addr.sa, &sz, SOCK_NONBLOCK | SOCK_CLOEXEC);
     con.fd       = fd;
 
     if (likely(fd >= 0)) {
+      // check for throttle
+      if (!opt.backdoor && check_net_throttle(ACCEPT)) {
+        // close the connection as we are in throttle state
+        con.close();
```

Comment:
accept() and close() immediately, it may generate many TIME_WAIT connections.
I'm not familiar with how the kernel processing these TIME_WAIT connections.
Please file a new issue or PR to discuss it.

My suggestion: we should also do `check_net_throttle(ACCEPT)` before schedule NetAccept in NetHandler.

Reply:
This will leave the connections in CLOSE_WAIT state. I can talk with kernel guys and if such a change is possible, we can change all the throttling code in a different PR.